WordPress/AbstractFunctionRestrictionsSniff.php

```diff
@@ -140,30 +140,27 @@ protected function setup_groups( $key ) {
 	 * @return void
 	 */
 	public function process( PHP_CodeSniffer_File $phpcsFile, $stackPtr ) {
-		$tokens = $phpcsFile->getTokens();
-		$token  = $tokens[ $stackPtr ];
+		$tokens        = $phpcsFile->getTokens();
+		$token         = $tokens[ $stackPtr ];
+		$token_content = strtolower( $token['content'] );
```

Comment:
The quite convoluted `if()` statement below it was triggering the `Squiz.PHP.DisallowMultipleAssignments` sniff (twice), this probably slipped in as I was looking to simplify the code.

Function names are case-insensitive in PHP, so should always be lowercased before comparisons are done.
I'm happy to move that bit to another PR (I will probably include it in one of the review PRs which I've mentioned in some recent issues in that case).